bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -1488,13 +1487,10 @@ impl Client {
     fn process_session_change(&self, session_change: SessionChange) {
         if let Some(delegate) = self.delegate.get().cloned() {
             debug!("Applying session change: {session_change:?}");
-            get_runtime_handle().spawn_blocking(move || match session_change {
-                SessionChange::UnknownToken { soft_logout } => {
+            get_runtime_handle().spawn_blocking(move || {
+                if let SessionChange::UnknownToken { soft_logout } = session_change {
```

Comment:
nit: let's keep a match statement, even if the `TokensRefreshed` arm is empty, that means that if we ever added another arm, we'd have to pause and think about what to do here (because the compiler would complain the match arm isn't handled)